### PR TITLE
An alternative explanation of cardinality

### PR DESCRIPTION
**Pull request by reyang**
@MrAlias suggested an [alternative explanation of cardinality](https://github.com/open-telemetry/opentelemetry-specification/pull/3856#discussion_r1473591297), trying to see which version would folks prefer.

### CODE REVIEW

**Review by MrAlias — APPROVED** [review]

File: specification/metrics/sdk.md [hunk 1]
```diff
@@ -732,9 +732,9 @@ of metrics across successive collections.
 **Status**: [Experimental](../document-status.md)
 
 SDKs SHOULD support being configured with a cardinality limit. The number of
-unique combinations of attributes is called cardinality. For a given metric, the
-cardinality limit is a hard limit on the number of metric points that can be
-collected during a collection cycle.
+unique attributes sets passed by a user is called cardinality. For a given
```
Comment (tsloughter) [inline specification/metrics/sdk.md:735]:
```suggestion
unique attribute sets passed by a user is called cardinality. For a given
```

File: specification/metrics/sdk.md [hunk 2]
```diff
@@ -732,9 +732,9 @@ of metrics across successive collections.
 **Status**: [Experimental](../document-status.md)
 
 SDKs SHOULD support being configured with a cardinality limit. The number of
```
Comment (yurishkuro) [inline specification/metrics/sdk.md:734]:
```suggestion
SDKs SHOULD support configurable cardinality limits. The number of
```

File: specification/metrics/sdk.md (hunk 1 above)
Comment (yurishkuro) [reply specification/metrics/sdk.md:735]: I would phrase it as "number of distinct permutations of attributes"

Comment (MrAlias) [reply specification/metrics/sdk.md:735]: @yurishkuro that would mean `a->a, b->b, c->c` and `c->c, a->a, b->b` would be distinct given a permutation implies order.

Comment (yurishkuro) [reply specification/metrics/sdk.md:735]: I was thinking of order of attribute names as fixed, but fair enough

File: specification/metrics/sdk.md [hunk 3]
```diff
@@ -731,10 +731,10 @@ of metrics across successive collections.
 
 **Status**: [Experimental](../document-status.md)
 
-SDKs SHOULD support being configured with a cardinality limit. The number of
-unique combinations of attributes is called cardinality. For a given metric, the
-cardinality limit is a hard limit on the number of metric points that can be
-collected during a collection cycle.
+SDKs SHOULD support configurable cardinality limits. The number of
+unique attribute sets passed by a user is called cardinality. For a given
```
Comment (trask) [inline specification/metrics/sdk.md:735]: "passed by a user" seems to imply before any metric attribute filters are applied? (e.g. https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument-advisory-parameter-attributes)

Comment (pellared) [reply specification/metrics/sdk.md:735]: I think that reopening and solving https://github.com/open-telemetry/opentelemetry-specification/pull/3803 could be helpful.

**Pull request closed without merging**